packages/browser-utils/test/browser-utils.test.ts

```diff
@@ -0,0 +1,30 @@
+import { isIOS, observeIosNavbar } from '../browser-utils';
+
+const { expect } = chai;
+const { sinon } = window;
+
+(isIOS ? describe : describe.skip)('iOS navbar', () => {
```

Comment:
I guess nothing but the `isIOS` property in `browser-utils` prevents these tests from passing on other than iOS devices. Should we add a flag to enable testing on any device?

Reply:
We could mock `navigator` too, to mimic the `iOS` and run all the tests. Will do.

Reply:
Changed to run the tests in all browsers by mocking `navigator`. Had to place the mock in a separate file because `isIOS` is only evaluated once, and otherwise it is too late.